README.md

```diff
@@ -17,6 +17,20 @@ $ npm start
 
 # generate static project
 $ npm run generate
-```
+
 
 For detailed explanation on how things work, checkout the [Nuxt.js docs](https://github.com/nuxt/nuxt.js).
+```
+## Translations
+
+Translations are pulled regularly from crowdin and commited in this repo. Please be aware that locales other than en_US will be overwritten in production.
+
+### Building
+docker build -f crowdin.dockerfile . -t crowdin-cd-com
```

Comment:
why not run `docker build -f crowdin.dockerfile . -t coderdojo/crowdin-cd-com` so you dont have to explicitly tag it later